students/PeterB/Session2/fizzbuzz.py

```diff
@@ -1,11 +1,8 @@
 for fizzbuzz in range(100):
     if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
         print("fizzbuzz")
-        continue
-    elif fizzbuzz % 3 == 0:
-        print("fizz")
-        continue
-    elif fizzbuzz % 5 == 0:
-        print("buzz")
-        continue
+    fizzbuzz % 3 == 0
```

Comment:
make sure to test the code! this prints a whole bunch too may fizzes and buzzes....

you didn't need the continue, but you DO need the `elif`s